components/bootloader_support/src/esp32/bootloader_esp32.c

```diff
@@ -211,6 +211,13 @@ esp_err_t bootloader_init(void)
     // print 2nd bootloader banner
     bootloader_print_banner();
 
+#ifdef CONFIG_ESP_SIMPLE_BOOT
```

Comment:
Is this specific to simple-boot? Could it be used in MCUboot init as well?

Reply:
It is used in mcuboot as well. 'CONFIG_ESP_SIMPLE_BOOT' is kept enabled for MCUBoot as well:
```c
config ESP_SIMPLE_BOOT
	bool "Simple Boot method"
	default y if !BOOTLOADER_MCUBOOT
	help
	  The Simple Boot is a method of booting that doesn't depend on a
	  2nd stage bootloader. Please note that some of the bootloader features
	  are not available using simple boot, such secure boot and OTA.
```

Perhaps we could check for `!BOOTLOADER_MCUBOOT` instead.

Reply:
> Perhaps we could check for !BOOTLOADER_MCUBOOT instead.

Maybe `config ESP_SIMPLE_BOOT` should be fixed to exclude `MCUBOOT`. But in this case it is better to use `!BOOTLOADER_MCUBOOT`.